test/Microsoft.AspNet.Razor.Test/TestFiles/CodeGenerator/Source/DynamicAttributeTagHelpers.cshtml

```diff
@@ -0,0 +1,14 @@
+﻿@addTagHelper "something, nice"
+
+<input unbound="prefix @DateTime.Now" />
```

Comment:
Are simple cases like `unbound="@variable"`, `unbound="literal"`, and `unbound=""` covered somewhere else?